docs/gadgets/trace/capabilities.md

```diff
@@ -142,6 +142,134 @@ You can now delete the pod you created:
 $ kubectl delete -f docs/examples/app-set-priority-locked-down.yaml
 ```
 
+#### Interpreting advanced columns
+
+Some columns are not displayed by default:
+* `caps`: the effective capability bitfield from the objective context of the process
+* `capsnames`: same as caps in a human friendly format
+* `currentuserns`: the user namespace of the process
+* `targetuserns`: the user namespace that the kernel used to test the
+  capability.
+
+They can be useful to understand advanced usage of capabilities.
+Let's see two examples.
+
+```
+$ kubectl run -ti --rm --restart=Never \
+    --image busybox --privileged testcaps -- \
+    chroot /
+```
+
+```
+$ kubectl gadget trace capabilities \
+    -o custom-columns=comm,syscall,capName,verdict,targetuserns,currentuserns,caps,capsnames
+COMM             SYSCALL                      CAPNAME            VERDICT TARGETUSERNS        CURRENTUSERNS       CAPS                 CAPSNAMES
+chroot           chroot                       SYS_CHROOT         Allow   4026531837          4026531837          3fffffffff           chown,dac_override,dac_…
```

Comment:
In a future PR, I´d like to switch cap names like `SYS_CHROOT` to lower case to be consistent with other columns.